src/app.ts

```diff
@@ -1,15 +1,54 @@
 import express = require("express");
```

Comment:
I did not notice it also in my own commits but we are using import and not required.

Reply:
Yeah I've seen the ide notifying me that it might change it to import - I think import is a relatively new concept and it's moving away from require into import -- not sure though, worth a google search